### ISSUE TITLE
Were the models finetuned?

### ISSUE THREAD

**Issue by amd-sonsingh** [thunlp/Ouroboros#3]
Great work and thanks for open-sourcing the implementation.

I have a quick question: while reporting the accuracy metrics in your paper (Tables 5 and 6), did you run inference on the models directly or did you first fine-tune them?

Thanks in advance!

**Comment by huangyuxiang03** [comment]
We inference on the model directly. Ouroboros is a train-free algorithm, thus no training is conducted in this research. Thanks for asking!
